test/new-e2e/agent-subcommands/flare/flare_test.go

```diff
@@ -27,6 +28,9 @@ func TestFlareSuite(t *testing.T) {
 }
 
 func requestAgentFlareAndFetchFromFakeIntake(v *commandFlareSuite, flareArgs ...client.AgentArgsOption) flare.Flare {
+	// Wait for the fakeintake to be ready to avoid 503 when sending the flare
+	// Definitely not the best way to do it, but the easiest at the moment
+	time.Sleep(60 * time.Second)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Any reason not to use an eventually (eg. eventually some basic metric is received by the fakeintake) ?

Reply:
Was easier this way, but yes the solution you described is way better. I'll change that when the framework will have a function to easily check that the fakeintake is ready

Reply:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can test that the `fakeintake` is ready by checking you get a `StatusOK` response on `v.Env().Fakeintake.Client.GetServerHealth()` --> https://github.com/DataDog/datadog-agent/blob/main/test/fakeintake/client/client.go#L148-L158

You could do something like 

```golang
assert.EventuallyWithT(t, func(c *assert.CollectT) {
    assert.NoErr(c, v.Env().Fakeintake.Client.GetServerHealth())
}) 
```